tests/vip-helpers/test-vip-utils.php

```diff
@@ -1,6 +1,20 @@
 <?php
 
 class WPCOM_VIP_Get_Resized_Attachment_Url_Test extends \WP_UnitTestCase {
```

Comment:
Should the tests go in a new class? The name of this class implies it's just for `Get_Resized_Attachment_Url` :)

Reply:
Maybe we should add a `tests/vip-helpers/vip-utils` directory to contain the individual parts? `vip-utils.php` is pretty big and unrelated.

Reply:
Good call, done in d80ecdaf

Reply:
Splitting into smaller test files is a great idea.